### PR TITLE
feat(ecs): add iops and throughput to disk

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/ecs/ TESTARGS='-run=TestAccComputeInstance_diskIopsThroughput'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ecs/ -v -run=TestAccComputeInstance_diskIopsThroughput -timeout 360m -parallel 4
=== RUN   TestAccComputeInstance_diskIopsThroughput
=== PAUSE TestAccComputeInstance_diskIopsThroughput
=== CONT  TestAccComputeInstance_diskIopsThroughput
--- PASS: TestAccComputeInstance_diskIopsThroughput (247.46s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ecs       247.485s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/ecs/resource_huaweicloud_compute_instance_test.go [hunk 1]
```diff
@@ -594,3 +627,40 @@ resource "huaweicloud_compute_instance" "test" {
...
+
+  network {
+    uuid              = data.huaweicloud_vpc_subnet.test.id
+    source_dest_check = false
+  }
+
+  system_disk_type = "GPSSD2"
+  system_disk_size = 50
+  system_disk_iops = 3000
+  system_disk_throughput = 125
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/ecs/resource_huaweicloud_compute_instance_test.go:650]: align equal signs

File: docs/resources/compute_instance.md [hunk 2]
```diff
@@ -353,8 +377,20 @@ The `network` block supports:
...
-* `type` - (Required, String, ForceNew) Specifies the ECS data disk type, which must be one of available disk types,
-  contains of *SSD*, *GPSSD* and *SAS*. Changing this creates a new instance.
+* `type` - (Required, String, ForceNew) Specifies the ECS data disk type. Changing this creates a new instance.
+
+  For details about disk types, see
+  [Disk Types and Disk Performance](https://support.huaweicloud.com/en-us/productdesc-evs/en-us_topic_0014580744.html).
+  Available options are:
+  + `SAS`: high I/O disk type.
+  + `SSD`: ultra-high I/O disk type.
+  + `GPSSD`: general purpose SSD disk type.
```
Comment (houpeng80) [inline docs/resources/compute_instance.md]:
```suggestion
  + `SAS`: high I/O disk type.
  + `SSD`: ultra-high I/O disk type.
  + `GPSSD`: general purpose SSD disk type.
```
change the first letter to upper case of the first word in the description

File: huaweicloud/services/acceptance/ecs/resource_huaweicloud_compute_instance_test.go (hunk 1 above)
Comment (niuzhenguo) [reply huaweicloud/services/acceptance/ecs/resource_huaweicloud_compute_instance_test.go:650]: done

File: docs/resources/compute_instance.md (hunk 2 above)
Comment (niuzhenguo) [reply docs/resources/compute_instance.md]: done

**Pull request merged**